lib/plugins/features/movies/controller.js

```diff
@@ -34,3 +37,25 @@ exports.list = async (query) => {
 
   }).fetchAll({ withRelated: ['locations'] });
 };
+
+exports.addLocationToMovie = async (movie_id, payload) => {
+  const movie = await new Movie({ id: movie_id }).fetch({ withRelated: ['locations'], require: true })
+  .catch(Error, () => {
+    throw Boom.notFound('could not find movie');
+  });
+
+  const location = await new Location({ name: payload.locationName }).fetch({ require: true })
+  .catch(Error, async () => {
```

Comment:
Since this is just a general error type, you can remove the `Error` parameter and just include the function as the single param to `.catch()`.